js/ad_manager_vast.js

```diff
@@ -1299,7 +1299,8 @@ OO.Ads.manager(function(_, $) {
         tracking: {},
         // clickTracking needs to be remembered because it can exist in wrapper ads
         clickTracking: filterEmpty($(linearXml).find("ClickTracking").map(function() { return $(this).text(); })),
-        clickThrough: filterEmpty($(linearXml).find("ClickThrough").map(function() { return $(this).text(); })),
+        //There can only be one clickthrough as per Vast 2.0/3.0 specs and XSDs
+        clickThrough: $(linearXml).find("ClickThrough").text(),
```

Comment:
find() returns a jQuery object which contains all the found elements. If there is nothing found, the jQuery object will still be returned, but it will contain no elements.